### PR TITLE
Implement design for Sharezone Plus lock screen on "information sheet ready by" page 

### LINKED ISSUE #884: Block read by page for information sheet for non Sharezone Plus users
https://www.figma.com/file/EIlLPe7KdF4bQXLQVAsDeM/SharezoneApp?type=design&node-id=2379-5225&mode=design&t=XAyp3Z0WOjT8vZ58-4

For the background can just dummy users instead of loading the real list.

### PR DESCRIPTION
**Pull request by nilsreichardt**
## Demo

image

## Related Tickets

Closes #884

### CODE REVIEW

File: app/test_goldens/blackboard/details/blackboard_item_read_by_users_list_page_test.dart [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+        ),
+        wrapper: materialAppWrapper(theme: theme),
+      );
+    }
+
+    group('renders sharezone plus ad as expected', () {
+      setUp(() {
+        when(subscriptionService.hasFeatureUnlocked(
+                SharezonePlusFeature.infoSheetReadByUsersList))
+            .thenAnswer((_) => true);
```
Comment (Jonas-Sander) [inline app/test_goldens/blackboard/details/blackboard_item_read_by_users_list_page_test.dart:79]:
```suggestion
            .thenAnswer((_) => false);
```

File: app/test_goldens/blackboard/details/blackboard_item_read_by_users_list_page_test.dart [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+          'blackboard_item_read_by_users_list_page_sharezone_ad_dark',
+        );
+      });
+    });
+
+    group('renders list as expected', () {
+      setUp(() {
+        when(subscriptionService.hasFeatureUnlocked(
+                SharezonePlusFeature.infoSheetReadByUsersList))
+            .thenAnswer((_) => false);
```
Comment (Jonas-Sander) [inline app/test_goldens/blackboard/details/blackboard_item_read_by_users_list_page_test.dart:105]:
```suggestion
            .thenAnswer((_) => true);
```

**Review by Jonas-Sander — APPROVED** [review]
These two values are wrong. You will have regenerate the goldens.  
Otherwise LGTM 👍

**Comment by nilsreichardt** [comment]
Good catch, thnks 👍

**Pull request merged**